src/pages/collections/[collectionId]/index.tsx

```diff
@@ -30,6 +31,7 @@ export const getServerSideProps = createServerSideProps({
 export default function Collections({ collectionId }: { collectionId: number }) {
   return (
     <CollectionsLayout>
+      <Meta title={`Collection | ${collectionId}`} description="View the collection details" />
```

Comment:
This Meta would be replaced with the one inside the `Collection` component used down below, no need for it here
```suggestion
```
We might need to truncate the description on that Meta though, can you please update it accordingly?

It should be something like the following
```jsx
<Meta
  // ...
  description={truncate(removeTags(collection.description), { length: 150 })}
/>
```